lib/cli/commands/cancelallorders.ts

```diff
@@ -0,0 +1,21 @@
+import { Arguments, Argv } from 'yargs';
+import { CancelAllOrdersRequest, CancelAllOrdersResponse } from '../../proto/xudrpc_pb';
+import { callback, loadXudClient } from '../command';
+
+export const command = 'cancelallorders';
+
+export const describe = 'removes all orders';
+
+export const builder = (argv: Argv) => argv
+    .example('$0 cancelallorders', describe);
+
+const formatOutput = (response: CancelAllOrdersResponse.AsObject) => {
+  response.removedOrdersList.forEach((removedOrder => console.log(`Cancelled order with id ${removedOrder}`)));
+  response.failedOrdersList.forEach((failedOrder => console.log(`Failed to cancel order with id ${failedOrder}`)));
+
+  console.log('No more orders found');
```

Comment:
One more afterthought, I don't think we need this message here, since it's a synchronous call that returns immediately. I also think we should handle the case where no orders are canceled, writing "no more orders found" when no orders were found at all doesn't make sense. Instead I think we should print "no orders found" when nothing was canceled, and just the canceled order ids without "no more orders found" when we do have orders that were canceled.